code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -0,0 +1,264 @@
+
+	assailant.changeNext_move(CLICK_CD_RANGE)
+
+	assailant.flick_attack_overlay(victim, "grab")
+
+	log_combat(assailant, victim, "psychically grabbed")
+	msg_admin_attack("[key_name(assailant)] psychically grabbed [key_name(victim)]" )
+
+	succeed_activate()
+	add_cooldown()
+
+
+// ***************************************
+// *********** Psychic Cure
+// ***************************************
+/datum/action/xeno_action/activable/psychic_cure
+	name = "Psychic Cure"
+	action_icon_state = "heal_xeno"
+	mechanics_text = "Heal and remove debuffs from a target."
+	cooldown_timer = 1 MINUTES
+	plasma_cost = 200
+	keybind_signal = COMSIG_XENOABILITY_PSYCHIC_CURE
+
+
+/datum/action/xeno_action/activable/psychic_cure/on_cooldown_finish()
+	to_chat(owner, "<span class='notice'>We gather enough mental strength to cure sisters again.</span>")
+	return ..()
+
+
+/datum/action/xeno_action/activable/psychic_cure/can_use_ability(atom/target, silent = FALSE, override_flags)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(QDELETED(target))
+		return FALSE
+	var/dist = get_dist(owner, target)
+	switch(dist)
+		if(-1)
+			if(!silent && target == owner)
+				to_chat(owner, "<span class='warning'>We cannot cure ourselves.</span>")
+			return FALSE
+		if(0 to 3)
+			if(!owner.line_of_sight(target))
+				to_chat(owner, "<span class='warning'>We can't focus properly without a clear line of sight!</span>")
+				return FALSE
+		if(4 to INFINITY)
+			if(!silent)
+				to_chat(owner, "<span class='warning'>Too far, our mind power does not reach it...</span>")
+			return FALSE
+	if(!isxeno(target))
+		return FALSE
+	var/mob/living/carbon/xenomorph/patient = target
+	if(!CHECK_BITFIELD(use_state_flags|override_flags, XACT_IGNORE_DEAD_TARGET) && patient.stat == DEAD)
+		if(!silent)
+			to_chat(owner, "<span class='warning'>It's too late. This sister won't be coming back.</span>")
+		return FALSE
+
+
+/datum/action/xeno_action/activable/psychic_cure/use_ability(atom/target)
+	if(owner.action_busy)
+		return FALSE
+
+	if(!do_mob(owner, target, 1 SECONDS, BUSY_ICON_FRIENDLY, BUSY_ICON_MEDICAL))
+		return FALSE
+
+	GLOB.round_statistics.psychic_cures++
+	owner.visible_message("<span class='xenowarning'>A strange psychic aura is suddenly emitted from \the [owner]!</span>", \
+	"<span class='xenowarning'>We cure [target] with the power of our mind!</span>")
+	target.visible_message("<span class='xenowarning'>[target] suddenly shimmers in a chill light.</span>", \
+	"<span class='xenowarning'>We feel a sudden soothing chill.</span>")
+
+	playsound(target,'sound/effects/magic.ogg', 75, 1)
+	new /obj/effect/temp_visual/telekinesis(get_turf(target))
+	var/mob/living/carbon/xenomorph/patient = target
+	patient.heal_wounds(SHRIKE_CURE_HEAL_MULTIPLIER)
+	if(patient.health > 0) //If they are not in crit after the heal, let's remove evil debuffs.
+		patient.SetKnockedout(0)
+		patient.SetStunned(0)
+		patient.SetKnockeddown(0)
+		patient.stagger = 0

```

Comment:
set_staggered? (also if being pedantic the names should follow the same pattern)

Reply:
The issue is that the variable is `knocked_down` there, for example.
I can change it to `SetStagger`, though I don't feel it's the best pattern.

Reply:
Personally what I would do is make the new ones use the SetStuff naming pattern, then make a separate PR renaming all instances of these (since there's a lot) if you really hate how it's done.

Reply:
Done.